input/docs/cs/affine_picture_transformation.md

```diff
@@ -0,0 +1,34 @@
+﻿title: Afinní transformace obrázku pomoci rotace
+--- 
+Transformační matice T je matice o velikosti 3x3, která se používá k provedení operace posunu, rotaci a odražení objektů v afinní transformaci 2D obrázků vůči původním souřadnicím osy.
```

Comment:
Tahle věta je užitečná. Nicméně je špatně napsaná, (pojem "odražení objektů" v tomto smyslu neexistuje). Zjisti si co je to transformační matice a jak se používá (už jsi to nějak psala, má to co dělat s násobením matic). Napiš to zcela jednoduše, bez akademického nádechu, tak aby to bylo zcela jasné. Aby z následujícího kódu bylo poznat jak to s tím souvisí.